data/json/items/gun/shot.json

```diff
@@ -522,8 +523,8 @@
     "copy-from": "shotgun_base",
     "looks_like": "remington_870",
     "type": "GUN",
-    "name": { "str": "Saiga shotgun" },
-    "description": "This shotgun is chambered in 12 gauge, and accepting Saiga magazines.",
+    "name": { "str": "Saiga tactical shotgun" },
```

Comment:
This has a brand name because it has corresponding magazines, if you were wondering.